tests/test_pdo.py

```diff
@@ -197,9 +168,14 @@ def test_pdos_refresh_rate(motion_controller, refresh_rate):
         mc.capture.pdo.start_pdos(COMMUNICATION_TYPE.Ethercat, refresh_rate)
 
 
-@pytest.mark.soem
-def test_start_pdos(connect_to_all_slaves):
-    mc, aliases = connect_to_all_slaves
+@pytest.mark.soem_multislave
+@pytest.mark.smoke
```

Comment:
I'm not sure if this is really a smoke test but at least there should be a multislave + smoke test for the tests execution to not fail